modules/serverless-create-default-channel.adoc

```diff
@@ -0,0 +1,47 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-channels.adoc
+
+[id="serverless-create-default-channel_{context}"]
+= Creating a channel with cluster default configuration
+
+To create a channel using the cluster default configuration set by the cluster administrator, you must create a generic `Channel` custom object.
+
+.Example `Channel` object
+[source,yaml]
+----
+apiVersion: messaging.knative.dev/v1beta1
+kind: Channel
+metadata:
+  name: example-channel
+  namespace: default
+----
+
+When the above `Channel` object is created, a mutating admission webhook adds a set of `spec.channelTemplate` properties for the `Channel` object based on the default channel implementation chosen by the cluster administrator.
+
+.Example `Channel` object with `spec.channelTemplate` properties
+[source,yaml]
+----
+apiVersion: messaging.knative.dev/v1beta1
+kind: Channel
+metadata:
+  name: example-channel
+  namespace: default
+spec:
+  channelTemplate:
+    apiVersion: messaging.knative.dev/v1beta1
+      kind: InMemoryChannel
```

Comment:
yaml formatting error,  the last "kind: InMemoryChannel" should be on the same level as the "apiVersion: messaging.knative.dev/v1beta1" above it.